files/common.sh

```diff
@@ -36,6 +36,8 @@ fi
 _tmp="$(mktemp)"
 exec 2>>"$_tmp"
 
+unset LANG LC_ALL
```

Comment:
interesting, what is LANG afterwards? Could we explicitly set LC_ALL=C?

Reply:
`LANG` is unset after that :upside_down_face:.

I was not sure about forcing `LC_ALL` to `C` or `C.UTF-8`, then recalled that older FreeBSD versions do not support `C.UTF-8` and that if the user used something else that UTF-8 with extended ASCII forcing UTF-8 would still break it.  But maybe we can do better: we only *need* to change `LC_MESSAGES` to have messages in english, and for `LC_*`, the value is:
* The value of `LC_ALL` if set (highest priority);
* The value of `LC_*` if set;
* The value of `LANG` (lowest priority).

So, we can:
1. Copy `LC_ALL` in `LANG` if `LANG` is unset;
2. Unset `LC_ALL`
3. Set `LC_MESSAGES` to C

```
export LANG=${LANG:-$LC_ALL}
unset LC_ALL
export LC_MESSAGES=C
```

This should preserve a correct behavior for users of accented chars.

Reply:
I adjusted the PR accordingly. I guess no packages have a name with accented char so this would not have caused trouble in the tasks of this module, but something makes me think this code should be part of Bolt itself and be run before running any shell scripts on remote systems…